Dockerfile

```diff
@@ -1,8 +1,28 @@
-FROM ruby:2.5
+FROM ruby:2.5.6
 
-# Install gems into a temporary directory
-COPY Gemfile* ./
-RUN gem install bundler && bundle install
+WORKDIR /var/www/inferno
 
-# Expose the port
+### Install dependencies
+
+COPY Gemfile* /var/www/inferno/
+RUN gem install bundler
+# Throw an error if Gemfile & Gemfile.lock are out of sync
+RUN bundle config --global frozen 1
+RUN bundle install
+
+### Install Inferno
+
+RUN mkdir data
+COPY public /var/www/inferno/public
+COPY resources /var/www/inferno/resources
+COPY config* /var/www/inferno/
+COPY Rakefile /var/www/inferno/
+COPY test /var/www/inferno/test
```

Comment:
Is this directory copied over so we can do testing on the Docker image?

Reply:
Yes. I'd really prefer to keep all the tests and testing dependencies out of it, but no good solutions for that came to mind. Something to keep in mind for the future.